GoMFT script fails with [ERROR] in line 45

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

GoMFT

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/gomft.sh)"

### 📝 Provide a clear and concise description of the issue.

GoMFT script fails with [ERROR] in line 45

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Run bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/gomft.sh)" in Proxmox shell

### ❌ Paste the full error output (if available).

   ______      __  _______________
  / ____/___  /  |/  / ____/_  __/
 / / __/ __ \/ /|_/ / /_    / /   
/ /_/ / /_/ / /  / / __/   / /    
\____/\____/_/  /_/_/     /_/     
                                  
  ⚙️  Using Default Settings on node proliant
  🖥️  Operating System: debian
  🌟  Version: 12
  📦  Container Type: Unprivileged
  💾  Disk Size: 4 GB
  🧠  CPU Cores: 1
  🛠️  RAM Size: 512 MiB
  🆔  Container ID: 128
  🚀  Creating a GoMFT LXC using the above default settings
  
  ✔️  Using Backup for Template Storage.
  ✔️  Using local-lvm for Container Storage.
  ✔️  Updated LXC Template List
  ✔️  LXC Template is ready to use.
  ✔️  LXC Container 128 was successfully created.
  ✔️  Started LXC Container
  ✔️  Set up Container OS
  ✔️  Network Connected: 10.0.0.137 
  ✔️  IPv4 Internet Connected
  ✖️  IPv6 Internet Not Connected
  ✔️  DNS Resolved github.com to 140.82.121.3
  ✔️  Updated Container OS
  ✔️  Installed Dependencies
  ✔️  Setup Golang
 ⠼ Setup GoMFT 
[ERROR] in line 45: exit code 0: while executing command wget -q "https://github.com/StarFleetCPTN/GoMFT/releases/download/v${RELEASE}/gomft-v${RELEASE}-linux-amd64" -O gomft
root@proliant:~# 

### 🖼️ Additional context (optional).

_No response_

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -40,8 +40,10 @@ function update_script() {
         wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
         tar -xzf $temp_file
         cp -rf GoMFT-${RELEASE}/* /opt/gomft
+        cd /opt/gomft
+        $STD go mod download
+        $STD go build -o gomft
         $STD $HOME/go/bin/templ generate
-        wget -q "https://github.com/StarFleetCPTN/GoMFT/releases/download/v${RELEASE}/gomft-v${RELEASE}-linux-amd64" -O /opt/gomft/gomft
         chmod +x /opt/gomft/gomft
         echo "${RELEASE}" >/opt/${APP}_version.txt
         msg_ok "Updated $APP to ${RELEASE}"

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -35,16 +35,17 @@ set -o pipefail
 msg_ok "Setup Golang"
 
 msg_info "Setup ${APPLICATION}"
-temp_file2=$(mktemp)
+temp_file=$(mktemp)
 RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
-wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file2
-tar -xzf $temp_file2
+wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
+tar -xzf $temp_file
 mv GoMFT-${RELEASE}/ /opt/gomft
 cd /opt/gomft
-$STD go install github.com/a-h/templ/cmd/templ@latest
-wget -q "https://github.com/StarFleetCPTN/GoMFT/releases/download/v${RELEASE}/gomft-v${RELEASE}-linux-amd64" -O gomft
-$STD $HOME/go/bin/templ generate
+$STD go mod download
+$STD go build -o gomft
 chmod +x gomft
+$STD go install github.com/a-h/templ/cmd/templ@latest
+$STD $HOME/go/bin/templ generate
 JWT_SECRET_KEY=$(openssl rand -base64 24 | tr -d '/+=')
 
 cat <<EOF >/opt/gomft/.env
@@ -93,7 +94,7 @@ motd_ssh
 customize
 
 msg_info "Cleaning up"
-rm -f $temp_file $temp_file2
+rm -f $temp_file
 $STD apt-get -y autoremove
 $STD apt-get -y autoclean
 msg_ok "Cleaned"

